Automattic/wp-calypso
# Odie: update store after new event

## Proposed Changes

* Fix rendering error from the textarea.
* Update the store after adding a new event.

## Why are these changes being made?

Currently we are creating multiple Odie chats because the store is not reflecting the new odie ID. This causes us to create new chat ID's thinking there is no current ID.

More context here - p1731738318382859-slack-C05FNGX7SKX

## Testing Instructions

1. Clear chat and refresh
2. Open HC and start a new odie chat
3. Send 2 or three messages
4. Check the odie chats - `[ mission-control ]/odie/odie-chats/` and make sure you only started one chat
5. Refresh and make sure its all good still
